### DISCUSSION TITLE
My website at https://su8.github.io/ is not working

### DISCUSSION THREAD

**Discussion by su8** [community/community#134836] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I just recovered my account after exchanging email accounts and found out that my website https://su8.github.io/ is not working. What did changes are necessary to make it working again ?

Thanks in advance.

**Comment by davevad93** [comment, marked as answer]
Hi @su8 ,

It doesn't look like you have any active deployments, go to the repo `Settings` => `Pages` and build and deploy your project. After doing so you should be able to access your website correctly.

**Reply by su8** [reply]
Thank you very much. I just clicked "static website" and magically my website is alive, thank you very much.

**Reply by davevad93** [reply]
You're welcome @su8 👍

**Reply by luckygoswami** [reply]
where is this option?
